nodejs/week1/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Nodeprojects",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
```

Comment:
You need to add two scripts here, according to exercise:
start will run the command node src/backend/index.js. This we use when we want to start up the server.
dev will run the command nodemon src/backend/index.js. This we use when we want to develop our application.